addons/logger_debug/models/models.py

```diff
@@ -0,0 +1,33 @@
+
+import logging
+from odoo import models, fields, api
+from odoo.addons.logger_debug.tools.log_filter_debug import get_log_filter_debug_db
+
+_logger = logging.getLogger(__name__)
+
+
+class LoggerDebug(models.Model):
+    _name = 'logger_debug.logger_debug'
+    _description = 'Switch log level to DEBUG to any odoo module'
+
+    @staticmethod
+    def _debug_module(module_name: str):
+        """Debug a module by setting the log level to DEBUG."""
+        module_logger = logging.getLogger(module_name)
+        module_logger.setLevel(logging.DEBUG)
```

Comment:
move this to the add_logger logic. This command is only needed if it is the first database to set in debug